### PR TITLE
Add package description file

### PR DESCRIPTION
**Pull request by etiennebacher**
https://github.com/easystats/easystats/issues/314

---

test failures unrelated to this PR

### CODE REVIEW

File: R/modelbased-package.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#' \code{modelbased}
+#'
+#' @title modelbased: Estimation of Model-Based Predictions, Contrasts and Means
+#'
+#' @description
+#'
+#' `modelbased` is a package helping with model-based estimations, to
+#' easily compute of marginal means, contrast analysis and model predictions.
+#'
+#'
```
Comment (IndrajeetPatil) [inline R/modelbased-package.R:10]:
```suggestion
```

File: R/modelbased-package.R [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#' @docType package
+#' @aliases modelbased modelbased-package
+#' @name modelbased-package
+#' @keywords internal
+"_PACKAGE"
```
Comment (IndrajeetPatil) [inline R/modelbased-package.R:15]:
```suggestion
"_PACKAGE"

```

**Comment by IndrajeetPatil** [comment]
Note that similar changes also need to be made for other PRs where files are missing the end-of-file (EOF) character.

**Comment by etiennebacher** [comment]
Ok I think it should be fine on other PRs now

**Review by IndrajeetPatil — APPROVED** [review]
Thanks!

**Pull request merged**